plugin-modernizer-core/src/main/java/io/jenkins/tools/pluginmodernizer/core/github/GHService.java

```diff
@@ -0,0 +1,122 @@
+package io.jenkins.tools.pluginmodernizer.core.github;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.api.errors.GitAPIException;
+import org.eclipse.jgit.api.errors.RefAlreadyExistsException;
+import org.eclipse.jgit.transport.RefSpec;
+import org.eclipse.jgit.transport.UsernamePasswordCredentialsProvider;
+import org.kohsuke.github.GHIssueState;
+import org.kohsuke.github.GHPullRequest;
+import org.kohsuke.github.GHRepository;
+import org.kohsuke.github.GitHub;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@SuppressFBWarnings(value = "PATH_TRAVERSAL_IN", justification = "false positive")
+public class GHService {
+
+    private static final Logger LOG = LoggerFactory.getLogger(GHService.class);
+
+    private static final String GITHUB_TOKEN = System.getenv("GITHUB_TOKEN");
```

Comment:
Why not on settings ? It would be better to centralise it here and pass it to contructor